### PR TITLE
fix: np-opto correction

### LINKED ISSUE #59: Fix indentation error in NP-opto correction (results are not affected)
**Describe the bug**
Due to an indendation error in the NP-opto correction, the settings files in an OpenEphys folder are retreived 384 times..This doesn't affect the result.

**To Reproduce**
See `correct_np_opto_electrode_locations` function: https://github.com/AllenNeuralDynamics/nd-data-transfer/blob/main/src/transfer/util/npopto_correction.py#L21-L28

**Expected behavior**
Settings files are retieved only once

### PR DESCRIPTION
**Pull request by alejoe91**
Closes #59 

And pre-calculates correct NP-opto locations

### CODE REVIEW

File: src/transfer/util/npopto_correction.py [hunk 1]
```diff
@@ -9,23 +9,23 @@
 Y_PITCH = 20
 NUMEL_IN_COL = 192
 
+npopto_electrode_xpos_arr = [X_OFFSET, X_OFFSET + X_PITCH] * NUMEL_IN_COL
```
Comment (dyf) [inline src/transfer/util/npopto_correction.py:12]: can we wrap this in a function?  helps keep the global namespace clean and gives us something to test.  I think `npopto_electrode_xpos` and `npopto_electrode_ypos` are all you need below, so:

```
NPOPTO_ELECTRODE_XPOS, NPOPTO_ELECTROD_YPOS = doit(...)
```

should do it?

**Review by dyf — CHANGES_REQUESTED** [review]

File: src/transfer/util/npopto_correction.py (hunk 1 above)
Comment (alejoe91) [reply src/transfer/util/npopto_correction.py:12]: Yep, will do it now

Comment (alejoe91) [reply src/transfer/util/npopto_correction.py:12]: Can I leave the constant variables outside or should they also be defined inside the function?

Comment (dyf) [reply src/transfer/util/npopto_correction.py:12]: I would leave whatever variables you need to be global outside and nothing else.

**Comment by jtyoung84** [comment]
@alejoe91 I merged in a github action that checks that the version number is being incremented. It's not perfect, but we have to manually change the version number in transfer/__init__.py on every pull request. (The current version in main is 0.0.4, so it needs to be bumped to 0.0.5)

**Comment by alejoe91** [comment]
Ok, is it necessary to bump up at every PR? Then is the package automatically deployed on pypi?

**Comment by alejoe91** [comment]
> @alejoe91 I merged in a github action that checks that the version number is being incremented. It's not perfect, but we have to manually change the version number in transfer/**init**.py on every pull request. (The current version in main is 0.0.4, so it needs to be bumped to 0.0.5)

done

**Comment by jtyoung84** [comment]
> Ok, is it necessary to bump up at every PR? Then is the package automatically deployed on pypi?

I'm gonna add the publishing to pypi step next week.

**Review by dyf — APPROVED** [review]

**Pull request merged**